dev/NavigationView/NavigationView.cpp

```diff
@@ -1594,6 +1734,222 @@ void NavigationView::UpdateVisualStateForDisplayModeGroup(const winrt::Navigatio
     }
 }
 
+void NavigationView::OnNavigationViewItemTapped(const winrt::IInspectable& sender, const winrt::TappedRoutedEventArgs& args)
+{
+    if (auto nvi = sender.try_as<winrt::NavigationViewItem>())
+    {
+        if (IsSettingsItem(nvi))
+        {
+            OnSettingsInvoked();
+        }
+        else
+        {
+            OnNavigationViewItemInvoked(nvi);
+        }
+        nvi.Focus(winrt::FocusState::Pointer);
+    }
+}
+
+void NavigationView::OnNavigationViewItemKeyUp(const winrt::IInspectable& sender, const winrt::KeyRoutedEventArgs& args)
```

Comment:
>OnNavigationViewItemKeyUp [](start = 21, length = 25)

Does design prefer this to occur in OnKeyDown? If so could we use OnPreviewKeyDown?

Reply:
That seems like a good change to make, however in order to try get this PR in sooner, I think its a good idea to merge the current implementation for now. Should be alright since this is how the Settings item behaves, which is a working implementation of a ListViewItem that’s been living out of the ListView. I'll keep track of this in a separate issue.

Reply:
https://github.com/microsoft/microsoft-ui-xaml/issues/1835